edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/CrdWatcher.cs

```diff
@@ -223,7 +223,7 @@ private async Task UpsertDeployments(V1ServiceList currentServices, V1Deployment
                     // Default labels
                     var labels = new Dictionary<string, string>
                     {
-                        [Constants.K8sEdgeModuleLabel] = KubeUtils.SanitizeLabelValue(moduleId.ModuleId),
+                        [Constants.K8sEdgeModuleLabel] = deploymentName,
```

Comment:
What's the difference between module id and deployment name?

Reply:
moduleId is module identity. 
deployment name is module name sanitized to k8s name rules.